src/harness/reference_models/move_list_calc/move_list.py

```diff
@@ -0,0 +1,392 @@
+        constraint: 	protection constraint, a tuple with named fields of
+                        'latitude', 'longitude', 'low_frequency', 'high_frequency'
+        num_iteration:  a number of Monte Carlo iterations
+
+    Returns:
+        interference: 	a dictionary containing CBSD index, median interference contribution,
+                        K random interference contributions of the grant to the protection
+                        constraint c, and bearing from c to CBSD location.
+    """
+
+    # Get CBSD grant information 	
+    grant_index = cbsd_grant.get('grantIndex')
+    lat_cbsd = cbsd_grant.get('latitude')
+    lon_cbsd = cbsd_grant.get('longitude')
+    h_cbsd = cbsd_grant.get('height')
+    indoor_cbsd = cbsd_grant.get('indoorDeployment')
+    ant_beamwidth_cbsd = cbsd_grant.get('antennaBeamwidth')
+    ant_az_cbsd = cbsd_grant.get('antennaAzimuth')
+    maxEirpPerMHz_cbsd = cbsd_grant.get('maxEirp')
+    low_freq_cbsd = cbsd_grant.get('lowFrequency')
+    high_freq_cbsd = cbsd_grant.get('highFrequency')
+
+    # Get protection constraint information (protection point and frequency range)
+    lat_c = constraint.latitude
+    lon_c = constraint.longitude
+    low_freq_c = constraint.low_frequency
+    high_freq_c = constraint.high_frequency
+    h_c = REF_INC_ANTENNA_HT
+
+    # Compute median and K random realizations of path loss/interference contribution
+    # based on ITM model as defined in [R2-SGN-03] (in dB)
+    reliabilities = np.random.rand(num_iteration)    # get K random reliability values from an
+                                                     # uniform distribution over [0,1)
+    reliabilities = np.append(reliabilities, [0.5])  # add 0.5 (for median loss) as
+                                            # a last value to reliabilities array
+    results = wf_itm.CalcItmPropagationLoss(lat_cbsd, lon_cbsd, h_cbsd, lat_c, lon_c, h_c,
+                                            indoor_cbsd,
+                                            reliability=reliabilities,
+                                            freq_mhz=FREQ_PROP_MODEL)
+    path_loss = np.array(results.db_loss)
+
+    # Compute CBSD EIRP inside the frequency range of protection constraint
+    eirpPerMHz_cbsd = antenna.GetStandardAntennaGains(results.incidence_angles.hor_cbsd,
+                                                      ant_az_cbsd, ant_beamwidth_cbsd,
+                                                      maxEirpPerMHz_cbsd)
+    eff_bandwidth = min(high_freq_cbsd, high_freq_c) - max(low_freq_cbsd, low_freq_c)
+    eirp_cbsd = eirpPerMHz_cbsd + 10 * np.log10(eff_bandwidth / 1000000.0)
+
+    # Calculate the interference contributions
+    interf = eirp_cbsd - path_loss
+    median_interf = interf[-1]      # last element is the median interference
+    K_interf = interf[:-1]          # first 'K' interference
+
+    # Store interference contributions
+    interference = {
```

Comment:
Again I recommend namedtuple rather than dict for simple records

Reply:
We will have a namedtuple 'InterferenceContribution' in the updated script.